deployment/pcm/templates/podmonitor.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.podMonitor }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "pcm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
```

Comment:
It would be handy if you could add custom labels to the `PodMonitor` to facilitate various filtering options provided by prometheus-operator.

Reply:
done and documented [here](https://github.com/intel/pcm/pull/727#issuecomment-2101116347)